[INFO   ] Logger: Record log in /storage/emulated/0/Projects/kivyapp2.0/.kivy/logs/kivy_25-04-01_71.txt
[INFO   ] Kivy: v2.2.1
[INFO   ] Kivy: Installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/__init__.py"
[INFO   ] Python: v3.11.4 (main, Sep 30 2023, 10:54:38) [GCC 11.4.0]
[INFO   ] Python: Interpreter at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/bin/python3"
[INFO   ] Logger: Purge log fired. Processing...
[INFO   ] Logger: Purge finished!
[INFO   ] Factory: 190 symbols loaded
[INFO   ] KivyMD: 1.2.0, git-Unknown, 2025-03-29 (installed at "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivymd/__init__.py")
[WARNING] KivyMD: Version 1.2.0 is deprecated and is no longer supported. Use KivyMD version 2.0.0 from the master branch (pip install https://github.com/kivymd/KivyMD/archive/master.zip)
[INFO   ] Image: Providers: img_tex, img_dds, img_sdl2, img_pil (img_ffpyplayer ignored)
[INFO   ] Text: Provider: sdl2
[INFO   ] Window: Provider: sdl2
[INFO   ] GL: Using the "OpenGL ES 2" graphics system
[INFO   ] GL: Backend used <sdl2>
[INFO   ] GL: OpenGL version <b'OpenGL ES 3.2 v1.r32p1-01eac0.efd03ef21f136842c0935bd4f493fe81'>
[INFO   ] GL: OpenGL vendor <b'ARM'>
[INFO   ] GL: OpenGL renderer <b'Mali-G77 MC9'>
[INFO   ] GL: OpenGL parsed version: 3, 2
[INFO   ] GL: Texture max size <16383>
[INFO   ] GL: Texture max units <16>
[INFO   ] Window: auto add sdl2 input provider
[INFO   ] Window: virtual keyboard not allowed, single mode, not docked
[INFO   ] Clipboard: Provider: android
[INFO   ] GL: NPOT texture support is available
[WARNING] Base: Unknown <android> provider
[INFO   ] Base: Start application main loop
[INFO   ] Base: Leaving application in progress...
[WARNING] stderr: Traceback (most recent call last):
[WARNING] stderr:   File "/storage/emulated/0/Projects/kivyapp2.0/main.py", line 20, in <module>
[WARNING] stderr:     MainApp().run()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/app.py", line 956, in run
[WARNING] stderr:     runTouchApp()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 574, in runTouchApp
[WARNING] stderr:     EventLoop.mainloop()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 339, in mainloop
[WARNING] stderr:     self.idle()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 383, in idle
[WARNING] stderr:     self.dispatch_input()
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 334, in dispatch_input
[WARNING] stderr:     post_dispatch_input(*pop(0))
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/base.py", line 302, in post_dispatch_input
[WARNING] stderr:     wid.dispatch('on_touch_up', me)
[WARNING] stderr:   File "kivy/_event.pyx", line 731, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivymd/uix/behaviors/ripple_behavior.py", line 404, in on_touch_up
[WARNING] stderr:     return super().on_touch_up(touch)
[WARNING] stderr:            ^^^^^^^^^^^^^^^^^^^^^^^^^^
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/uix/behaviors/button.py", line 179, in on_touch_up
[WARNING] stderr:     self.dispatch('on_release')
[WARNING] stderr:   File "kivy/_event.pyx", line 727, in kivy._event.EventDispatcher.dispatch
[WARNING] stderr:   File "kivy/_event.pyx", line 1307, in kivy._event.EventObservers.dispatch
[WARNING] stderr:   File "kivy/_event.pyx", line 1191, in kivy._event.EventObservers._dispatch
[WARNING] stderr:   File "/data/user/0/ru.iiec.pydroid3/files/aarch64-linux-android/lib/python3.11/site-packages/kivy/lang/builder.py", line 55, in custom_callback
[WARNING] stderr:     exec(__kvlang__.co_value, idmap)
[WARNING] stderr:   File "/storage/emulated/0/Projects/kivyapp2.0/card.kv", line 12, in <module>
[WARNING] stderr:     on_release: root.manager.current="login"
[WARNING] stderr: ^^^^^^^^^^^^
[WARNING] stderr:   File "kivy/weakproxy.pyx", line 32, in kivy.weakproxy.WeakProxy.__getattr__
[WARNING] stderr: AttributeError: 'CustomCard' object has no attribute 'manager'
